### PR TITLE
Protect our configuration.

### PR DESCRIPTION
**Pull request by WouterSioen**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -17,6 +17,7 @@ FileETag MTime Size
 	RewriteRule composer\.json - [F]
 	RewriteRule composer\.lock - [F]
 	RewriteRule .*\.md - [F]
+	RewriteRule Capfile - [F]
```
Comment (tijsverkoyen) [inline .htaccess]: Eventueel ook nog `RewriteRule .*\.rb - [F]` toevoegen, dan kan er geen enkele rb-file gedownload worden

Comment (tijsverkoyen) [inline .htaccess]: En als ge dan toch bezig zijt:

    RewriteRule Gemfile - [F]
    RewriteRule Gemfile\.lock - [F]
    RewriteRule Gruntfile.coffee - [F]
    RewriteRule Sumofile - [F]
    RewriteRule composer - [F]
    RewriteRule composer.json - [F]
    RewriteRule package.json - [F]

**Comment by WouterSioen** [comment]
added

File: .htaccess (hunk 1 above)
Comment (tijsverkoyen) [inline .htaccess]: Dus eigenlijk:

    RewriteRule [Cap|Gem|Sumo]file(\.lock)? - [F]

**Pull request merged**